src/components/enterprise-user-subsidy/enterprise-offers/data/utils.js

```diff
@@ -7,8 +6,14 @@ import {
   ENTERPRISE_OFFER_TYPE,
 } from './constants';
 
-export const offerHasBookingsLimit = offer => offer.maxDiscount !== null || offer.maxUserDiscount !== null;
-export const offerHasEnrollmentsLimit = offer => offer.maxGlobalApplications !== null;
+export const offerHasBookingsLimit = offer => (
+  // requires loose equality check to account for both null and undefined
+  offer.maxDiscount != null || offer.maxUserDiscount != null
```

Comment:
Could use [_.isNil](https://lodash.com/docs/4.17.15#isNil) to check for null || undefined